src/globus_sdk/_testing/data/compute/submit.py

```diff
@@ -0,0 +1,41 @@
+from globus_sdk._testing.models import RegisteredResponse, ResponseSet
+
+from ._common import (
+    ENDPOINT_ID,
+    FUNCTION_ID,
+    TASK_ARGS,
+    TASK_ARGS_2,
+    TASK_GROUP_ID,
+    TASK_ID,
+    TASK_ID_2,
+)
+
+REQUEST_ID = "5158de19-10b5-4deb-9d87-a86c1dec3460"
+
+SUBMIT_RESPONSE = {
+    "request_id": REQUEST_ID,
+    "task_group_id": TASK_GROUP_ID,
+    "endpoint_id": ENDPOINT_ID,
+    "tasks": {
+        FUNCTION_ID: [TASK_ID, TASK_ID_2],
+    },
+}
+
+RESPONSES = ResponseSet(
+    metadata={
+        "endpoint_id": ENDPOINT_ID,
+        "function_id": FUNCTION_ID,
+        "task_id": TASK_ID,
+        "task_id_2": TASK_ID_2,
+        "task_args": TASK_ARGS,
+        "task_args_2": TASK_ARGS_2,
```

Comment:
That's correct. I'll move them directly to the test to clarify things.

Reply:
I'll wait for this and then intend to re-approve and merge.

I think it's slightly easier to read if these are just baked into the relevant test, since they aren't used for request matching or anything.